src/libraries/System.Formats.Tar/tests/TarEntry/TarEntry.ExtractToFile.Tests.cs

```diff
@@ -92,5 +94,68 @@ public void ExtractToFile_Link_Throws(TarEntryFormat format, TarEntryType entryT
 
             Assert.Equal(0, Directory.GetFileSystemEntries(root.Path).Count());
         }
+
+        public static IEnumerable<object[]> GetFormatsAndFiles()
+        {
+            foreach (TarEntryFormat format in new[] { TarEntryFormat.Ustar, TarEntryFormat.Pax, TarEntryFormat.Gnu })
```

Comment:
Any particular reason why V7 is being excluded? I don't see any posix or gnu fields being used in the tests below.

Reply:
I think they don't support the `TarEntryType.RegularFile, TarEntryType.Directory` types?

Reply:
I'll try and see what happens if I add it.